app/workers/question_test_worker.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'open3'
+
+class QuestionTestWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :test, retry: 1
+
+  def perform(submit_id, mission_id, question_id)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/AbcSize: Assignment Branch Condition size for perform is too high. [16.79/15]